ddl/table.go

```diff
@@ -152,6 +138,74 @@ func onCreateTable(d *ddlCtx, t *meta.Meta, job *model.Job) (ver int64, _ error)
 	}
 }
 
+func getBundleFromDBInfo(t *meta.Meta, job *model.Job, dbInfo *model.DBInfo) (*placement.Bundle, error) {
```

Comment:
Is this function necessary? The table will inherit placement options from database by default, I think the behavior is 'copy' the placement meta from database to  table and the later alter for database will not effect the database's tables. So just use getBundleFromTblInfo when creating a new table which inherits placement options from database?